### ISSUE TITLE
[Feature] Run time next to ratings underneath widgets on homescreen

### ISSUE THREAD

**Issue by JFG1000** [jurialmunkey/skin.arctic.horizon.2#930]
### Skin section

Home Widgets

### Feature description

![20230529_101639](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/76907836/a9d09e48-3c91-4cae-bbbe-5848375ca5a5)

Hi there would it please be possible to add a setting which enables us users to have the run time displayed next to the ratings, I've attached a photo 

### Why is this feature needed? Please justify and explain.

Would he great to see this instead of bringing up the info screen 

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by illuvattarr** [comment]
Yes, would appreciate this very much. Along with an 'Ends at' time. It is currently displaying runtime if you change expand information to full in stead of plot, but then you won't see the ratings...

**Comment by VelimirSaban** [comment]
Yes, please. Just yesterday I wrote on the KODI forum that I would like this setting.

**Comment by jurialmunkey** [comment]
If you enable full expand and change the setting from simple to fullscreen then you will see all the info.

**Comment by Gabba35** [comment]
> @jurialmunkey yes I'm aware of that but quite honestly don't like to use that on my home screen, if you could create a setting to allow us to display it, that would be much appreciated. Many thanks

This would be an amazing option, I'd love to use AH2 with this. End time is important to me and if it's late and a long movie/show I might not work it out so well in my head!

**Comment by VelimirSaban** [comment]
Any news on this? I'll be happy to buy you another beer for this. Please? :)
